src/main/java/frc/trigon/robot/utilities/mechanisms/ArmElevatorMechanism2d.java

```diff
@@ -0,0 +1,137 @@
+package frc.trigon.robot.utilities.mechanisms;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import org.littletonrobotics.junction.Logger;
+
+/**
+ * A Mechanism2d object to display the current angle and target angle, and the current position and target position of an arm elevator.
+ */
+public class ArmElevatorMechanism2d {
+    private static final double TARGET_POSITION_LIGAMENT_WIDTH = 10;
+    private final String key;
+    private final Mechanism2d mechanism;
+    private final MechanismLigament2d
+            currentPositionLigament,
+            targetPositionLigament;
+
+    public ArmElevatorMechanism2d(String key, double maximumDisplayablePosition) {
+        this(key, maximumDisplayablePosition, MechanismConstants.BLUE);
+    }
```

Comment:
No need for the default constructor. We'll want to always input a color since each mechanism will have a different one.